Error on Building GraalVM Native Image with Recent Update to v1.18.1

### Bug Description

In the last update, the -H:+BuildReport option was added to the [native-image.properties](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/commit/e0540592bf68045d1dbabaf217900833b9b8065e#diff-6e29181dc10fef682393ed6f7f0e4e24d054f94721cacf17ccc005c51b776a02) file. 

This presents an issue when trying to build a GraalVM native image using a GraalVM community edition. This option is not available in the CE. [build-reports docs](https://www.graalvm.org/jdk22/reference-manual/native-image/optimizations-and-performance/PGO/build-reports/)

Attempts to override the -H:+BuildReport option in the application's properties file (i.e., using quarkus.native.additional-build-args=-H:-BuildReport) have not addressed the problem.

### Example code (or command)

`./gradlew build -Dquarkus.package.type=native`

### Stacktrace

```
Error: Could not find option 'BuildReport' from 'jar:file:///[build-folder]/lib/com.google.cloud.sql.jdbc-socket-factory-core-1.18.1.jar!/META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/native-image.properties'. Use -H:PrintFlags= to list all available options.
> Task :quarkusAppPartsBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':quarkusAppPartsBuild'.
> There was a failure while executing work items
   > A failure occurred while executing io.quarkus.gradle.tasks.worker.BuildWorker
      > io.quarkus.builder.BuildException: Build failure: Build failed due to errors
                [error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
                at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:468)
                at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:258)
                at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
                at java.base/java.lang.reflect.Method.invoke(Method.java:580)
                at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:849)
                at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
                at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
                at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
                at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
                at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
                at java.base/java.lang.Thread.run(Thread.java:1583)
                at org.jboss.threads.JBossThread.run(JBossThread.java:483)

```

### Steps to reproduce?

1. Update com.google.cloud.sql.jdbc-socket-factory-core to `v1.18.1`
2. Use `21.0.2-graalce` or `22.0.1-graalce`
3. Run `./gradlew build -Dquarkus.package.type=native`


### Environment

1. OS type and version: macOS 14.4.1 (23E224)
2. Java SDK version: 21.0.2-graalce
3. Cloud SQL Java Socket Factory version: v1.18.1


### Additional Details

_No response_

--- a/core/src/main/resources/META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/native-image.properties
+++ b/core/src/main/resources/META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/native-image.properties
@@ -1,7 +1,6 @@
 Args =\
   -H:+UnlockExperimentalVMOptions \
   -H:+AddAllCharsets \
-  -H:+BuildReport \
   -H:ReflectionConfigurationResources=META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/jni-unix-socket-config.json \
   -H:ResourceConfigurationResources=META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/resource-config.json \
   -H:DynamicProxyConfigurationResources=META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/proxy-config.json \

